js/pages/trackers.es6.js

```diff
@@ -67,6 +60,14 @@ Trackers.prototype = $.extend({},
                 template: ItemMenuTemplate
             });
 
+            this.views.trackerlist = new TrackerListView({
+                pageView: this,
+                model: new TrackerListModel({heading: 'Top Blocked', max:5}),
```

Comment:
I realize now I shouldn't have used "heading" as an example model property, since we want to avoid strings in the javascript (for i18n reasons later on it will be better to keep strings in templates).